### ISSUE TITLE
favorite_places_category → favorite_places_and_tracks_category

### ISSUE THREAD

**Issue by beardhatcode** [osmandapp/OsmAnd#10244]
# 🚀 feature request

### Description

I have a few friends who I showed OsmAnd to, and they keep forgetting where GPX tracks can be managed. And I find myself repeating over and over that "it is under places".

### Describe the solution you'd like

Maybe "my places" should be renamed "places and tracks" in the sidebar.

### Describe alternatives you've considered

- We could also add a "Tracks" item to the sidebar (might be confused with directions)

**Comment by scaidermern** [comment]
"places and tracks" is pretty long. Maybe just "Personal" or "My Data" instead of "My Places"? However I'm not quite convinced here.

**Comment by vshcherb** [comment]
Yes, something like My Data is more generic

**Comment by beardhatcode** [comment]
I agree that "Places and tracks" is quite long.

"My Data" might be too generic. I don't know if I would go to "My data" if I was looking for a list of places I added to favorites.

I like "Personal" but it is not quite perfect in my opinion.

Maybe I (or someone else) will come up with the perfect wording next week.

**Comment by beardhatcode** [comment]
A Friend recommended the following:

> - Remove *Directions*, *Search* and *Configure map*: these options are aleady present on the card
> - Split "_My places_" in "_My places_" and "_My tracks_"
> - Rename "_Download maps_" to "_Manage maps_"
> - Place "_OsmAnd Live_" below "_Manage maps_"

This seems like an improvement to me.

**Comment by sonora** [comment]
Yes in deed, it is quite a historically grown mess: We have
- **Functionality** (like Search, Directions, Plan a route)
- **Settings** (all kinds, see #9047
- **Data management** (Manage maps, OsmAnd Live)
- **Personal data** (favs, tracks, notes)

But no real structure where to find what: We disperse items all over the map screen, the drawer, and different menu items.

I have long believed we should introduce some structure in the UI as follows:

- The functionality items each deserve to be shown on the top level, because they are the app's core use and accessed frequently.
- All other items should be moved under the 3 categories and only their 3 category headers are shown top level.

Side note regarding tracks: A particular issue is also that we have a second screen to access your tracks under "Configure maps/Tracks". Many users only find and know that screen, but it has many issues and is less powerful than MyPlaces/Tracks. See #1435

**Comment by NotSoImportant** [comment]
menu structure discussion is also here: https://github.com/osmandapp/OsmAnd/issues/9047

**Comment by sonora** [comment]
@NotSoImportant Thanks for taking the effort to mention and crosslink here and elsewhere a number of suggestions I had posted in recent years to simplify and streamline OsmAnd's UI, discoverability, and simplicity for everyday and also just occasional use.

OsmAnd has become quite a powerful tool containing a large variety of both basic (applicable to the general public of users) and extremely advanced functionality (applicable to a small number of experts).

I am convinced that someone who uses OsmAnd very frequently and for their specific purposes will get along just fine, but will have seized to notice how difficult it can be for the occasional user to get up to speed, or quickly perform a very basic operation,  "_Where is the Settings screen?_" or "_How do you stop an ongoing navigation?_" just being two illustrative questions.

Maybe I should take the time at some point to reference all these suggestions in one 'Super-Issue', or maybe Victor @vshcherb wants to think about a way to showcase a UI strategy, **maybe by collecting all related issues in a separate "UI ideas" pseudo-milestone** so we have better visibility and comparison?

**Comment by NotSoImportant** [comment]
Yes, having a more structured approach to streamline the discussions would help.

Completely agree that frequent users are fine (although still I keep getting confused some times), but new users are getting frustrated and deterred.

**Comment by GuentherGredy** [comment]
Let me leave a comment concerning UI look and feel, more concerning the look of a dialog, less the structure of the menus. Let me give you an example.
In the actual OsmAnd version I discovered a new dialog which can be reached by pressing "ALLE ANZEIGEN" of "Verlauf" in "Übersichtsseite". Now pressing one of the entries a longer time I get a new dialog (left part of the attached picture).
Opening "Meine Orte" I get another Dialog (right part of the attached picture).
As you can see, the controls presented in the two dialogs completely differ; sometimes there are just symbols, sometimes buttons with text, sometimes blue colors are used, sometimes it's just black.
I already suggested somewhere else to take the time and develop an OsmAnd Application Style Guide; this also would help making the learning curve steeper.
Thanks for your attention.
![StyleGuide](https://user-images.githubusercontent.com/85982948/151488394-1de03207-692e-4982-a647-d5b364e4fda4.png)
